apps/multitimer/app.js

```diff
@@ -328,9 +328,21 @@ function editTimer(idx, a) {
   setUI();
 }
 
+function readAndConvJson() {
+  let json = require("Storage").readJSON("multitimer.json", true);
```

Comment:
Probably missing a default value e.g. || {}

Reply:
Thank you!